<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="northstar">
        <include file="$(find tj2_northstar)/launch/ov2311.launch">
            <arg name="camera_info_url" default="package://tj2_bringup/config/diffyjr/northstar/calibration/camera_0.yaml"/>
            <arg name="camera_name" default="camera_0"/>
            <arg name="camera_index" default="0"/>
        </include>
        <include file="$(find tj2_northstar)/launch/ov2311.launch">
            <arg name="camera_info_url" default="package://tj2_bringup/config/diffyjr/northstar/calibration/camera_1.yaml"/>
            <arg name="camera_name" default="camera_1"/>
            <arg name="camera_index" default="1"/>
        </include>
    </group>
</launch>
